From 1dae7d6e8978c1d22a9846730dc5b966aef7a17a Mon Sep 17 00:00:00 2001
From: Mateus Nascimento <mateus.soaresdonascimento@windriver.com>
Date: Mon, 24 Mar 2025 14:21:33 -0300
Subject: [PATCH] Add custom pod annotations to libvirt

This change allows custom annotations to be added to the libvirt pods
using the static overrides configurations. It follows the recently made
commits of other OpenStack Helm pods.

---
 libvirt/templates/daemonset-libvirt.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libvirt/templates/daemonset-libvirt.yaml b/libvirt/templates/daemonset-libvirt.yaml
index 050aefc0..94d2ff05 100644
--- a/libvirt/templates/daemonset-libvirt.yaml
+++ b/libvirt/templates/daemonset-libvirt.yaml
@@ -65,6 +65,7 @@ spec:
 {{ tuple $envAll | include "helm-toolkit.snippets.release_uuid" | indent 8 }}
         configmap-bin-hash: {{ tuple "configmap-bin.yaml" . | include "helm-toolkit.utils.hash" }}
         configmap-etc-hash: {{ tuple "configmap-etc.yaml" . | include "helm-toolkit.utils.hash" }}
+{{ tuple "libvirt" . | include "helm-toolkit.snippets.custom_pod_annotations" | indent 8 }}
         configchecksum: {{ toYaml .Values.labels.isApplication | sha256sum | trunc 63 }}
     spec:
 {{ dict "envAll" $envAll "application" "libvirt" | include "helm-toolkit.snippets.kubernetes_pod_security_context" | indent 6 }}
-- 
2.34.1

